src/ui/components/atoms/danger-button/DangerButton.tsx

```diff
@@ -0,0 +1,13 @@
+import styles from './DangerButton.module.scss';
+import type { IProps } from './types/IProps';
+
+function DangerButton({ children, variant = 'danger-primary', onClick }: IProps) {
+  const variantClass = styles[`button-danger--${variant}`];
+  return (
+    <button className={`${styles['button-danger']} ${variantClass}`} onClick={onClick}>
```

Comment:
please change this approach, use this  https://github.com/Ditmar/backoffice-openscience/blob/master/src/ui/components/demo/calculator/atoms/display.tsx#L7-L10

Reply:
I have made the modifications requested in this PR:
https://github.com/Ditmar/backoffice-openscience/tree/feature/SEM-013/src/ui/components/atoms/danger-button